### ISSUE TITLE
Onion skin parameter accessibility from dockable panel

### ISSUE THREAD

**Issue by ohsnapitsjoel** [opentoonz/opentoonz#4526]
### Is your feature request related to a problem? Please describe

Hi! I'm really taking to OT. I'm actually more impressed with some of its features and the straightforward interface than I've been with Harmony, which I've used quite a bit.  

When it comes to onion skinning, I find that while it's generally very usable, it could be more robust.  I find that having to adjust the global opacity factor for onion skinning is a little cumbersome, and I don't get the opacity I'm hoping for when onion skinning multiple frames.  I also find that I often click the wrong relative frame in the timeline when trying to select frames for onion skinning.  

I also realize there isn't a way to onion skin actual drawings relative to the current drawing; only a number of frames relative to the timeline cursor is taken into account.   

### Describe the solution you'd like

I'd love to see the ability to control onion skin parameters like opacity, color, and relative frame selection on a more granular level from a dockable panel in the UI. I'm dreaming of something along the lines of Krita / Harmony / etc.:  
![image](https://user-images.githubusercontent.com/7570180/189184628-f9eb066c-b781-4aa8-8cac-1acc84faa239.png)

The ability to onion skin a number of _drawings relative to the current drawing_ as opposed to _numbers of frames relative to the current frame_ would also be fantastic.  Is there a workaround in OT to do something like this? Or could this be implemented?

**Comment by gab3d** [comment]
Agreed. I was just about to post a similar request for the mid/long term, along with a smaller one for the short term, to help improve the usefulness of onion skinning, when used in the inbetween creation stages of a new animation.

Regarding the above request, I do think a Light Table should be the ideal goal.
While is not shown in the screen capture, the "By Frames" dropdown menu also has a "By Drawings" alternative option, so the 1 2 3 ... 10 numbers could refer to either previous/next "Frames" or "Drawings", which is a possibility OT currently doesn't offer in its Onion Skin system.
When using the "By Drawings" option, the first shown drawing will be the next/previous exposed drawing that is different from the current one (i.e. it doesn't take into account held or implicit frames). That way, no matter how far the previous/next key frame is, it will always be shown if the "1" slider is up. 
The "2" slider refers to the second previous/next different exposed drawing in the layer, and so on...

This is very useful when creating a new animation, where you are more likely to just start spacing exposed drawings appart to be able to draw new inbetweens, while at the same time you may also start tweaking the position of those drawings in the timeline to give them the correct timing.

Currently, as you can only show either absolute timeline frames or frames relative to the current frame, you cannot really make good use of Relative onion skinning while creating an animation because:
A. you don't really know when the next key frame will happen in time.
B. not every keyframe will be spaced at the same distance everytime.

Then, Absolute onion skinning can be used if you mark the timeline frames that contain the specific key frames to be shown. But, at the same time, this has the disadvantage of having to be continually adjusting the marked frames, every time you move one of your previous/next key frames for whatever reason.

So, the possible ways I envision to solve that problem would be to either:
1. (In depth) Implement a Light Table panel, such as the one shown above, including the option to use a "By Drawings" mode.
2. (Workaround) To make OT move absolute marked onion skin frames together, when using the Shift + Drag function to move an exposed drawing. As Shift + Drag was implemented with this workflow (creating an animation from zerio in OT) in mind, it makes sense to check if there's a marked absolute onion skin frame at the same frame of the drawing being moved, and also move it with it.

Obviously the Light Table solution would be fantastic and I completely support it.
